packages/firebase_analytics/firebase_analytics/lib/src/firebase_analytics.dart

```diff
@@ -125,7 +125,15 @@ class FirebaseAnalytics extends FirebasePluginPlatform {
     );
   }
 
-  /// Sets the applicable end user consent state. 'default' value for 'adStorageConsentGranted' & 'analyticsStorageConsentGranted' is 'true'
+  /// Sets the applicable end user consent state.
+  /// By default, no consent mode values are set. 
+  ///
+  /// Default consents can be set according to the platform: 
+  /// - [iOS][1]
+  /// - [Android][2]
+  ///
+  /// [1]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=ios#default-consent
+  /// [2]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=android#default-consent
```

Comment:
According to the comment here, native APIs needs to be adjusted.  
@russellwheatley  I know that you are internally discussing this, so if you think I can continue this PR let me know and I can adjust it, if not, you can just close the PR.

Reply:
May I message you on discord? @russellwheatley